remove app beta testing text from documentation (#207)
microbiomedata/docs: content/home/src/howto_guides/submit2nmdc.md

# Submitting to the NMDC

## Introduction: Making metadata capture easy
The NMDC has developed two integrated resources for improving metadata capture: the [NMDC Submission Portal](https://data.microbiomedata.org/submission/home) (released April, 2022) and the [NMDC Field Notes](https://microbiomedata.org/field-notes/) mobile app (released April 2025). Each resource was designed to lower barriers to capturing metadata and adhering to community standards, thereby addressing the critical gap of collecting necessary information describing a study, its biosamples, and associated assays.

### NMDC Submission Portal

> The NMDC Submission Portal was designed using a flexible framework leveraging a new modeling approach called the Linked Data Modeling Language (LinkML) and the template-driven spreadsheet tool, [DataHarmonizer](https://github.com/cidgoh/DataHarmonizer). It supports several different community standards, such as the _Minimum Information about any (x) Sequence_ (MIxS) standard from the Genomic Standards Consortium (GSC), the _PROV_ standard for provenance metadata, the _Proteomics Standards Initiative_ (PSI) standards for metaproteomics, and the _Metabolomics Standards Initiative_ (MSI) standards for metabolomics. The NMDC Submission Portal is an intuitive interface that allows researchers to provide information about their study, the metadata about the study’s biosamples, multi-omics data associated with the study, and whether Department of Energy (DOE) User Facility generated data and analysis outputs. Updates and new features are continually being implemented as user research provides new insights to improve usability, and as standards are updated and improved. 

Author: "The NMDC Submission Portal is an intuitive interface that allows researchers to provide information about their study, the metadata about the study’s biosamples, multi-omics data associated with the study, and whether Department of Energy (DOE) User Facility generated data and analysis outputs."

This sentance is long and ends weird. Suggested update in google doc

### NMDC Field Notes mobile app

> The NMDC Field Notes mobile app is designed to support real-time registration and collection of standardized metadata and field measurements. This mobile app is built following the same schema format as the Submission Portal and similarly supports the community standards [described above](### NMDC Submission Portal) as well as in [Collaboration to support Community Standards](### Collaboration to support Community Standards) below. When using the NMDC Field Notes mobile app, metadata can be captured for your samples while in the field using in-app tools and formatting. The NMDC Field Notes mobile app can be used offline and provides a direct connection to the NMDC Submission Portal so your metadata is already there upon field work completion.

Author: as above & below 

Is that weird?

### NMDC Field Notes mobile app
The NMDC Field Notes mobile app is designed to support real-time registration and collection of standardized metadata and field measurements. This mobile app is built following the same schema format as the Submission Portal and similarly supports the community standards [described above](### NMDC Submission Portal) as well as in [Collaboration to support Community Standards](### Collaboration to support Community Standards) below. When using the NMDC Field Notes mobile app, metadata can be captured for your samples while in the field using in-app tools and formatting. The NMDC Field Notes mobile app can be used offline and provides a direct connection to the NMDC Submission Portal so your metadata is already there upon field work completion.

#### Connecting NMDC Field Notes and the NMDC Submission Portal

> The NMDC Field Notes mobile app synchronizes data with the NMDC Submission Portal and vice versa. This means that changes made in one place will be reflected in the other. Additional details of this synchronization and important considerations are captured in the Field Notes mobile app [documentation and guide](https://github.com/microbiomedata/nmdc-field-notes/blob/main/src/pages/GuidePage/md/guide-page.md).

Reviewer: Ah I didn't realize you'd be linking to this Markdown file. I considered it to be more of a source code file than a documentation file. I'm not saying you have to change anything. This is mostly a note to myself that we need to be aware of this link if we ever want to move this file around in the source tree.

# Submitting to the NMDC

## Introduction: Making metadata capture easy

> The NMDC has developed two integrated resources for improving metadata capture: the [NMDC Submission Portal](https://data.microbiomedata.org/submission/home) (released April, 2022) and the [NMDC Field Notes](https://microbiomedata.org/field-notes/) mobile app (released April 2025). Each resource was designed to lower barriers to capturing metadata and adhering to community standards, thereby addressing the critical gap of collecting necessary information describing a study, its biosamples, and associated assays.

Author: "assays" feels very sequencing focused?
analyses? (Asked in google doc, might update)

## Introduction

> The NMDC has developed two products for improving metadata capture: the [NMDC Submission Portal](https://data.microbiomedata.org/submission/home) (released April, 2022) and the [NMDC Field Notes](https://microbiomedata.org/field-notes/) app. Each product was designed to lower barriers to capturing metadata and adhering to community standards, thereby addressing the critical gap of collecting necessary metadata describing a study and its biosamples.

Reviewer 2: Do we want to mention "(released March 2025)" for the Field Notes app ?

[![](../_static/images/howto_guides/portal_guide/ORCiD.png)](../_static/images/howto_guides/portal_guide/ORCiD.png)

NMDC resources require ORCiD authentication for full access and functionality.

## Submission Portal Functionality

[https://data.microbiomedata.org/submission/home](https://data.microbiomedata.org/submission/home) 

The Submission Portal requires ORCiD authentication to access. If you have already signed in via ORCID, you will not see this login screen within the Submission Portal.

[![](../_static/images/howto_guides/portal_guide/Create_submission.png)](../_static/images/howto_guides/portal_guide/Create_submission.png)

Once signed in with an ORCiD, you will see an option to ‘Create New Submission’ with subsequent guidance to provide information required for submission to the NMDC. Details about each section are outlined below. Submitters can return to existing submissions saved under the ORCiD account to resume their work at any point.   

### Submission Context

[![](../_static/images/howto_guides/portal_guide/submission_context.png)](../_static/images/howto_guides/portal_guide/submision_context.png)

> On the Submission Context screen, submitters are asked whether data has already been generated for their study.  If a user selects ‘Yes’ a DOI associated with the data can be entered. For data generated at either the JGI or EMSL, the specific Award DOI (e.g., 10.46936/10.25585/60001289) should be entered, along with selecting the checkbox for this option. If the data was not generated at a DOE User Facility, a valid data DOI can be entered. A data DOI is not the same as a publication DOI, and would be issued through a separate resource as a unique persistent identifier (e.g., 10.48443/e4zf-b917). 

Author: Inconsistent us of "you" or "submitter" or "users"

Pick one and make it consistent. I suggest "submitter"

Author: Will submit an issue and make this change in a later PR

Reviewer 2: +1 for "submitters" or "users". "you" feels a bit weird

## Resource Access

[![](../_static/images/howto_guides/portal_guide/portal_functionality.png)](../_static/images/howto_guides/portal_guide/portal_functionality.png)

All NMDC resources are accessible from the "Products" tab on https://microbiomedata.org/.

[![](../_static/images/howto_guides/portal_guide/ORCiD.png)](../_static/images/howto_guides/portal_guide/ORCiD.png)

> NMDC resources require ORCiD authentication for full access and functionality.

Reviewer: ```suggestion
NMDC resources require ORCID authentication for full access and functionality.
```

https://info.orcid.org/brand-guidelines/ The name of the organization is "ORCID". The thing they give you is an "ORCID iD".

## Submission Portal Functionality

[https://data.microbiomedata.org/submission/home](https://data.microbiomedata.org/submission/home) 

> The Submission Portal requires ORCiD authentication to access. If you have already signed in via ORCID, you will not see this login screen within the Submission Portal.

Reviewer: ```suggestion
The Submission Portal requires ORCID authentication to access. If you have already signed in via ORCID, you will not see this login screen within the Submission Portal.
```

[![](../_static/images/howto_guides/portal_guide/Create_submission.png)](../_static/images/howto_guides/portal_guide/Create_submission.png)

> Once signed in with an ORCiD, you will see an option to ‘Create New Submission’ with subsequent guidance to provide information required for submission to the NMDC. Details about each section are outlined below. Submitters can return to existing submissions saved under the ORCiD account to resume their work at any point.   

Reviewer: ```suggestion
Once signed in with an ORCID, you will see an option to ‘Create New Submission’ with subsequent guidance to provide information required for submission to the NMDC. Details about each section are outlined below. Submitters can return to existing submissions saved under the ORCID account to resume their work at any point.   
```

[![](../_static/images/howto_guides/portal_guide/contributors.png)](../_static/images/howto_guides/portal_guide/contributors.png)

> This includes listing Contributor names, ORCiDs, associating role(s) based on the CRediT (Contributor Roles Taxonomy), and their submission access permission. A Contributor can have a single or multiple roles.  

Reviewer: ```suggestion
This includes listing Contributor names, ORCID iDs, associating role(s) based on the CRediT (Contributor Roles Taxonomy), and their submission access permission. A Contributor can have a single or multiple roles.  
```

> Permission roles are granted by ORCiD and are as follows: 

Reviewer: ```suggestion
Permission roles are granted based on ORCID iD and are as follows: 
```

### Sample Metadata

The Sample Metadata interface consists of a grid in which each row represents one sample and each column represents one attribute of a Biosample. Users are provided with numerous convenience and organizational features (described below) to assist in metadata completion.

The NMDC uses sections for clarity, and to identify when MIxS specifications have been used as published by the GSC, or when the NMDC team has modified the description, examples, or validation rules for a MIxS attribute captured in their respective columns. These modifications are based on user research and feedback provided to the NMDC.

> Biosample metadata can be entered manually (by typing each row), or the data can be entered in bulk by importing a Microsoft Excel XLSX file. The metadata is updated each time a user navigates across the submission template without the need for manually clicking ‘save’. Upon completion, the study submission and metadata will be reviewed by a member of the NMDC team, and once approved, the submitting user will indicate when data are ready to be published to the Data Portal.

Reviewer 2: ```suggestion
Biosample metadata can be entered manually (by typing each row), or the data can be entered in bulk by importing a Microsoft Excel XLSX file. The metadata is updated each time a user navigates across the submission template, without the need for manually clicking ‘save’. Upon completion, the study submission and metadata will be reviewed by a member of the NMDC team, and once approved, the submitting user will indicate when data are ready to be published to the Data Portal.
```

> [![](../_static/images/howto_guides/portal_guide/find-replace.png)](../_static/images/howto_guides/portal_guide/find-replace.png)

Reviewer: I guess this screenshot (and probably some others, like the import/export one) needs to be updated with the UI reorganization that came with the metadata suggester. Not saying it needs to be done with these changes necessarily, but I just wanted to mention it.

Author: Yes.. I thought I had an issue... but apparently not! Making one now!

[https://microbiomedata.org/field-notes/](https://microbiomedata.org/field-notes/)

> The NMDC Field Notes app is for mobile device use. It runs on both Apple (iOS) and Android devices and is available in the app store.

Reviewer 2: "app store" -> "app stores" ?

After selecting the validate button, cells will become color coded to indicate invalid and incomplete metadata. Dark pink cells indicate a required cell has been left empty. Light pink indicates that there is an error in the formatting of the information entered into that cell. The "Column Information" tool, described above, provides expected structure patterns and examples of valid metadata, which you can refer to when addressing validation errors.

## NMDC Field Notes mobile app functionality

[https://microbiomedata.org/field-notes/](https://microbiomedata.org/field-notes/)

> The NMDC Field Notes app is for mobile device use. It runs on both Apple (iOS) and Android devices and is available in the app store.

Reviewer 2: ```suggestion
The NMDC Field Notes app is for mobile device use. It runs on both Apple (iOS) and Android devices and is available in the corresponding app stores.
```

This app is designed to streamline real-time registration and collection of standardized metadata and field measurements. 
Upon first use, each section of the app will automatically launch a quick tour. You can press the "→" button to continue the tour or press the "X" button to end it. To re-initialize the tours at any time, visit the Settings page and tap "Reset tours".
The "Guide" section of the app contains recommendations about fieldwork in general, and about collecting metadata for specific sample type(s). It also provides important considerations for synchronization with the Submission Portal. We recommend viewing it before going out into the field.

> The NMDC Field Notes mobile app requires ORCiD authentication to access it. 

Reviewer: ```suggestion
The NMDC Field Notes mobile app requires ORCID authentication to access it. 
```

> Submissions via the Field Notes mobile app are available in the Submission Portal as an automatic backup when connected to wifi. The app is available for offline use, but requires internet connection to **start** a submission. 

Reviewer: ```suggestion
Submissions created in the Field Notes mobile app are available in the Submission Portal. This sync happens automatically when connected to the internet. The app is available for offline use, but requires internet connection to **start** a submission. 
```

I'd avoid using the term "wifi" since the app doesn't make a distinction between a wifi and a cellular connection.

Submissions via Field Notes are available in the Submission Portal as an automatic backup when connected to wifi. The app is available for offline use, but requires internet connection to **start** a submission. 

[![](../_static/images/howto_guides/portal_guide/app-home-page.png)](../_static/images/howto_guides/portal_guide/app-home-page.png)

> ### Welcome to the NMDC Field Notes mobile app! 

Author: IDK why this said 1 of 3?

Reviewer 2: Excellent question :-D

[![](../_static/images/howto_guides/portal_guide/app-study-page.png)](../_static/images/howto_guides/portal_guide/app-study-page.png)

> The Study Information page requires a valid ‘Study Name’ and a valid email address. All other fields are optional and can be completed in the app or in the Submission Portal. Study names and all other information can be updated or added via the app or Submission Portal. See [Study](#study) for more details about Study information.

Reviewer: ```suggestion
The Study Information page requires a valid "Study Name" and Principal Investigator email address. All other fields are optional and can be completed in the app or in the Submission Portal. Study names and all other information can be updated or added via the app or Submission Portal. See [Study](#study) for more details about Study information.
```
